pkgs/development/python-modules/pydocstyle/default.nix

```diff
@@ -1,33 +1,40 @@
-{ lib, buildPythonPackage, fetchFromGitHub, isPy3k
-, mock
-, pytest
+{ lib
+, buildPythonPackage
+, isPy3k
+, fetchFromGitHub
 , snowballstemmer
+, pytestCheckHook
 }:
 
 buildPythonPackage rec {
   pname = "pydocstyle";
-  version = "5.0.2";
+  version = "6.1.1";
   disabled = !isPy3k;
 
+  format = "setuptools";
+
   src = fetchFromGitHub {
     owner = "PyCQA";
     repo = pname;
     rev = version;
-    sha256 = "03z8miyppm2xncrc9yjilwl7z5c5cpv51zha580v64p8sb2l0j7j";
+    hash = "sha256-j0WMD2qKDdMaKG2FxrrM/O7zX4waJ1afaRPRv70djkE=";
```

Comment:
I would prefer it to be sha256, I think a lot of the update scripts look for it

Reply:
confirmed, it does look for the `sha256` string

Reply:
changed it

Reply:
You mean `nix-update`? Works fine for me with `hash` instead of `sha256`.

Reply:
I meant `./maintainers/scripts/update-python-libraries`

Reply:
Turns out I was right that not yet everything supports hash.

Reply:
And we can fix that. What a time to be alive!

fwiw: fetchpatch also lacks support for `hash`.